Doc/using/mac.rst

```diff
@@ -2,190 +2,433 @@
+  paths and separate :file:`site-packages` directories so, by default,
+  if you need a package available in both builds, it may need to be installed in both.
+  The free-threaded package will install a separate instance of :program:`pip` for use
+  with |python_version_t|.
+
+  - To install a package using :command:`pip`` without a :command:`venv`:
+
+        |python_version_t| ``-m pip install _package_name_``
+
+- When working with multiple Python environments, it is usually safest and easiest
+  to :ref:`create and use virtual environments <tut-venv>`.
+  This can avoid possible command name conflicts and confusion about which Python is in use:
+
+      |python_version_t| ``-m venv directory_name``
+
+  then :command:`activate`.
+
+- To run a free-threaded version of IDLE:
+
+      |python_version_t| ``-m idlelib``
+
+- The interpreters in both builds respond to the same
+  :ref:`PYTHON environment variables <using-on-envvars>`
+  which may have unexpected results, for example, if you have ``PYTHONPATH``
+  set in a shell profile. If necessary, there are
+  :ref:`command line options <using-on-interface-options>` like ``-E``
+  to ignore these environment variables.
+
+- The free-threaded build links to the third-party shared libraries,
+  such as ``OpenSSL`` and ``Tk``, installed in the traditional framework.
+  This means that both builds also share one set of trust certificates
+  as installed by the :command:`Install Certificates.command` script,
+  thus it only needs to be run once.
+
+- If you cannot depend on the link in ``/usr/local/bin`` pointing to the
+  python.org free-threaded |python_version_t| (for example, if you want
+  to install your own version there or some other distribution does),
+  you can explicitly set your shell ``PATH`` environment variable to
+  include the ``PythonT`` framework ``bin`` directory:
+
+  .. code-block:: sh
+
+     export PATH="/Library/Frameworks/PythonT.framework/Versions/3.13/bin":"$PATH"
+
+  The traditional framework installation by default does something similar,
+  except for :file:`Python.framework`. Be aware that having both framework ``bin``
+  directories in ``PATH`` can lead to confusion if there are duplicate names
+  like ``python3.13`` in both; which one is actually used depends on the order
+  they appear in ``PATH``. The ``which python3.x`` or ``which python3.xt``
+  commands can show which path is being used. Using virtual environments
+  can help avoid such ambiguities. Another option might be to create
+  a shell :command:`alias` to the desired interpreter, like:
+
+  .. code-block:: sh
+
+     alias py3.13="/Library/Frameworks/Python.framework/Versions/3.13/bin/python3.13"
+     alias py3.13t="/Library/Frameworks/PythonT.framework/Versions/3.13/bin/python3.13t"
+
+Installing Free-threaded Binaries using the command line
+--------------------------------------------------------
+
+If you want to use automation to install the free-threaded interpreter
```

Comment:
Is there a reason to make this a free-threaded specific set of instructions? My inclination is that it's either something that is worth promoting to "normal" installs (since it's useful there, too); or not including at all (since, if you need to know about macOS installer automation, it's probably your job :-) )

Reply:
I reworded the section to indicate that it applies to all installs and use the previous free-threaded install as an example.